### PR TITLE
{lib}[foss/2021b] DeepLabCut v2.3.1 w/ Python 3.9.6 

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

Requires:
- [ ] PR #17550
- [ ] PR #16385 
- [x] PR #16873 
- [x] PR #16482

### CODE REVIEW

File: easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.1-foss-2021b-CUDA-11.4.1.eb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+]
+
+dependencies = [
+    ('CUDA', '11.4.1', '', SYSTEM),
```
Comment (boegel) [inline easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.1-foss-2021b-CUDA-11.4.1.eb:22]: Bump to `11.5.2`?

File: easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.1-foss-2021b-CUDA-11.4.1.eb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    ('PyYAML', '5.4.1'),
+    ('wxPython', '4.2.0'),
+    ('OpenCV', '4.5.5', local_version),
+    ('numba', '0.54.1'),
+    ('libyaml', '0.2.5'),
+    ('imgaug', '0.4.0', versionsuffix),
+    ('tqdm', '4.62.3'),
+    ('ruamel.yaml', '0.17.21'),
+    ('statsmodels', '0.13.1'),
+    ('PyTorch', '1.11.0', versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.1-foss-2021b-CUDA-11.4.1.eb:39]: bump to `1.12.1`, see #17272

File: easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.1-foss-2021b-CUDA-11.4.1.eb [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    ('MoviePy', '1.0.3'),
+    ('SciPy-bundle', '2021.10'),
+    ('HDF5', '1.12.1'),
+    ('LZO', '2.10'),
+    ('Blosc', '1.21.1'),
+    ('py-cpuinfo', '9.0.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/d/DeepLabCut/DeepLabCut-2.3.1-foss-2021b-CUDA-11.4.1.eb:45]: Some of these are dependencies for `PyTables`, are they also direct dependencies for `DeepLabCut` itself?

**Comment by sassy-crick** [comment]
Closed as PR https://github.com/easybuilders/easybuild-easyconfigs/pull/16385 was closed and also super-seeded by merged PR https://github.com/easybuilders/easybuild-easyconfigs/pull/19166

**Pull request closed without merging**